### PR TITLE
Increase the default memory for master and bootstrap nodes

### PR DESCRIPTION
**Pull request by yussufsh**
Signed-off-by: Yussuf Shaikh <yussuf.shaikh@ibm.com>

### CODE REVIEW

**Review by bpradipt — APPROVED** [review]
/lgtm

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/160#pullrequestreview-549005063" title="LGTM">bpradipt</a>*, *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/160#" title="Author self-approved">yussufsh</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ocp-power-automation%2Focp4-upi-powervs).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ocp-power-automation/ocp4-upi-powervs/blob/release-4.6/OWNERS)~~ [bpradipt,yussufsh]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: var.tfvars
Comment (bpradipt) [inline var.tfvars:16]: Let's change bootstrap to 32GB as well

**Review by bpradipt — CHANGES_REQUESTED** [review]
Let's change bootstrap to 32GB as well

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/160#" title="Author self-approved">yussufsh</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ocp-power-automation%2Focp4-upi-powervs).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ocp-power-automation/ocp4-upi-powervs/blob/release-4.6/OWNERS)~~ [yussufsh]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by ltccci** [comment]
New changes are detected. LGTM label has been removed.

**Comment by yussufsh** [comment]
> 
> Let's change bootstrap to 32GB as well

I have picked up the commit from PR #171

**Review by bpradipt — APPROVED** [review]
/lgtm

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/160#pullrequestreview-550932278" title="LGTM">bpradipt</a>*, *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/160#" title="Author self-approved">yussufsh</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ocp-power-automation%2Focp4-upi-powervs).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ocp-power-automation/ocp4-upi-powervs/blob/release-4.6/OWNERS)~~ [bpradipt,yussufsh]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by sudeeshjohn — APPROVED** [review]
/lgtm

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/160#pullrequestreview-550932278" title="LGTM">bpradipt</a>*, *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/160#pullrequestreview-551049782" title="LGTM">sudeeshjohn</a>*, *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/160#" title="Author self-approved">yussufsh</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ocp-power-automation%2Focp4-upi-powervs).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ocp-power-automation/ocp4-upi-powervs/blob/release-4.6/OWNERS)~~ [bpradipt,yussufsh]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
